src/Flex.php

```diff
@@ -757,18 +757,33 @@ public function fetchRecipes(array $operations): array
             }
 
             $noRecipe = !isset($manifests[$name]) || (isset($manifests[$name]['not_installable']) && $manifests[$name]['not_installable']);
-            if ($noRecipe && 'symfony-bundle' === $package->getType()) {
-                $manifest = [];
-                $bundle = new SymfonyBundle($this->composer, $package, $job);
-                if (null === $devPackages) {
-                    $devPackages = array_column($this->composer->getLocker()->getLockData()['packages-dev'], 'name');
+            if ($noRecipe) {
+                $bundles = [];
+
+                if ('symfony-bundle' === $package->getType()) {
```

Comment:
@nicolas-grekas What if we also change to `if ('symfony-bundle' === $package->getType() || isset($extra['symfony'])) {` when no explicit bundles are defined? So meaning as a fallback for the explicit bundles logic

Meaning any package with symfony in extra section is assumed to be a Symfony bundle as well for the auto detect logic?

Reply:
Hmm, maybe not, symfony section is used in many packages that are not bundles right?